guide/english/css/tutorials/index.md

```diff
@@ -12,4 +12,6 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
+If you have some basic experience with CSS and want to spice things up, you can try creating Pure CSS images. That's right. You're only going to be making pictures, logos, etc out of CSS. While it may not seem practical, it will get your creative juices flowing and help you get a better grasp on the language. The article below goes through the process and links to a free course on the topic.
```

Comment:
I suggest putting the url as an actual link before the tutorial summary as well as creating a valid link with link text using the syntax `[A Beginners Guide to Pure CSS Images](https://medium.com/coding-artist/a-beginners-guide-to-pure-css-images-ef9a5d069dd2)`
```suggestion
[A Beginners Guide to Pure CSS Images](https://medium.com/coding-artist/a-beginners-guide-to-pure-css-images-ef9a5d069dd2) - If you have some basic experience with CSS and want to spice things up, you can try creating Pure CSS images. That's right. You're only going to be making pictures, logos, etc out of CSS. While it may not seem practical, it will get your creative juices flowing and help you get a better grasp on the language. The article below goes through the process and links to a free course on the topic.
```